### PR TITLE
Simplifies Jenkinsfile

### PR DESCRIPTION
**Pull request by normanrz**
Refactors/simplifies Jenkinsfile to use our new [Jenkins library](https://github.com/scalableminds/jenkins-library)

### Steps to test:
- Still works: http://jenkins-library.webknossos.xyz/datasets/kevin_bad/view#144,105,56,0,2.00

------
- [x] Ready for review

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -1,188 +1,80 @@
...
+      docker-compose down --volumes --remove-orphans
+    """
+  }
 
-    } catch (err) {
 
-      currentBuild.result = "FAILURE"
-      echo err.toString()
-      throw err
+  dockerPublish { repo = "scalableminds/webknossos" }
```
Comment (jstriebel) [inline Jenkinsfile]: This seems to misuse the closure, just to set a variable. I'm changing it to a normal function with a single parameter.

Comment (normanrz) [reply Jenkinsfile]: Other libraries do use it that way, though.
https://github.com/fabric8io/jenkins-pipeline-library#deploy-project

Comment (jstriebel) [reply Jenkinsfile]: @normanrz Sorry, didn't see the comment in the outdated field.

Merged it already with the "normal" function that gets a map. I would argue for this version, as it contains the variable names as well (by accepting a map). Closures are fine if variables are dynamically set depending on the closure execution, but here I find normal functions much more intuitive.

If you prefer closures, we can also use them, but then also for `wrap` to have it consistent.

Comment (normanrz) [reply Jenkinsfile]: I think you are right. We should stick with functions. Thanks for reviewing and merging this PR :-)

**Pull request merged**